frontend/public/features.ts

```diff
@@ -128,11 +129,36 @@ const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
     }
   );
 
+const monitoringConfigMapPath = `${k8sBasePath}/api/v1/namespaces/monitoring/configmaps/sharing-config`;
```

Comment:
Does the namespace not have an `openshift-` prefix?

Reply:
Oops. Thanks.